main.js

```diff
@@ -0,0 +1,37 @@
+function makeCell() {
+    var divsize = 50;
+    var posx = (Math.random() * ($(".tissue-container").width() - divsize)).toFixed();
+    var posy = (Math.random() * ($(".tissue-container").height() - divsize)).toFixed();
+    $newCell = $("<div class='cell-clone'></div>").clone().css({
+        'left': posx + 'px',
+        'top': posy + 'px',
+        
+    });
+
+    $newCell.appendTo('.tissue-container');
+}
+
+
+function multiplyCell(){
+    for(var i = 0; i < 100; i++){
+        setTimeout(function timer(){
+            makeCell();
+        }, i * 65); 
+    }           
+}
+
+
+setTimeout( multiplyCell, 11000 );
```

Comment:
I would place this line (24) and the contents of 27-29 + 33-35 in a single `$(function() {` ... `});` block

Reply:
We have wrapped these 27-29 + 33-3 in one function but could you explain about why the set timeout function also  has to be wrapped in the same function. :)